include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -712,7 +698,7 @@ struct __result_and_scratch_storage_impl : __result_and_scratch_storage_base
     }
 
     virtual std::size_t
-    __get_data(sycl::event __event, std::size_t* __p_buf) const override
+    __get_data(sycl::event __event, std::size_t* __p_buf) override
```

Comment:
We remove `const` here because we calls `__get_value()` function which is non-cost now.

Reply:
`const` restored by declaring `sycl::queue` as `mutable`.